test/res.status.js

```diff
@@ -129,6 +129,20 @@ describe('res', function () {
           .expect(500, /Invalid status code/, done);
       });
 
+      it('should raise error for status code above 599 with strict status code', function (done) {
+        var app = express();
+
+        app.set('strict status codes', true);
+
+        app.use(function (req, res) {
+          res.status(600).end();
+        });
+
+        request(app)
+          .get('/')
+          .expect(500, /Status code must be greater than 99 and less than 600 in strict status codes./, done);
```

Comment:
should expect the error rather than a 500

Reply:
Do you mean verify against the RangeError and not the status code 500?

Reply:
`assert.throws()`

Reply:
my mistake -- this is good